src/app/features/fee-editor/stacks/stacks-fee-editor.provider.tsx

```diff
@@ -10,17 +12,19 @@ interface StacksFeeEditorProviderProps extends HasChildren {
   availableBalance: Money;
   marketData: MarketData;
   onGoBack(): void;
-  txOptions: StacksUnsignedTransactionOptions;
+  txOptions?: StacksUnsignedTransactionOptions;
+  unsignedTx?: StacksTransactionWire;
```

Comment:
Working on a better solution for this so we don't have to pass optional params. With this request we receive the unsignedTx rather than the options.